msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -46,17 +46,24 @@ You can create a new conversation with a user or a conversation thread in a chan
 * You can retrieve the [list of channels](~/bots/how-to/get-teams-context.md) in a team where your app is installed.
 * You can retrieve the [list of members](~/bots/how-to/get-teams-context.md) of a team where your app is installed.
 
-Regardless of how you get the information, store the `tenantId` and either the `userId` or `channelId` to create a new conversation. You can also use the `teamId` to create a new conversation thread in the general or default channel of a team.
+Regardless of how you get the information, store the `tenantId` and then store either the `aadObjectId`, `userId`, or `channelId` to create a new conversation. You can also use the `teamId` to create a new conversation thread in the general or default channel of a team. Ensure that the bot is installed in the chat or a team before you can send a proactive message to a channel or in a personal scope. 
```

Comment:
Installation is only required for proactive messaging in a personal scope using the user's AAD ID.